flang/unittests/Runtime/CUDA/CMakeLists.txt

```diff
@@ -1,15 +0,0 @@
-if (FLANG_CUF_RUNTIME)
```

Comment:
why was this file deleted?

Reply:
File has been moved to FortranRuntime/unittests/Runtime/CUDA/CMakeLists.txt (https://github.com/llvm/llvm-project/pull/110298/files/8b984fea71966e325e8e1306ab1da34be47ff746#diff-c6d68be4180e8ba3884bdd150b807e61c7b058d3a8a15683c916634b8e3dbca7). A copyright header has been added, which is apparently enough of a change that GitHub doesn't recognize the rename.